api/views/content.py

```diff
@@ -104,14 +105,23 @@ def retrieve(self, request, author_pk=None, pk=None):
         # Careful, gotchya here, if author_pk is none, it means we are dealing with
         # /author/:id and that the author id is going to be in pk
         if author_pk is None:
-            author = get_object_or_404(Author, id=pk)
-            data = AuthorSerializer(author).data
 
-            # append author posts to this object
-            posts = Post.objects.filter(author__id=pk)
-            if posts is not None:
-                posts = PostSerializer(posts, many=True)
-                data["posts"] = posts.data
+            # check if we're querying for a remote author
+            if "HTTP_AUTH_HOST" in request.META:
```

Comment:
Okay, updated to match. It performs a regex on an all caps version of the headers provided.